lua/entities/gmod_wire_digitalscreen/cl_init.lua

```diff
@@ -248,17 +248,17 @@ function ENT:RedrawPixel(a)
 		cr, cg, cb = (transformcolor[colormode] or transformcolor[0])(c)
 	end
 
-	local xstep = (512/self.ScreenWidth)
-	local ystep = (512/self.ScreenHeight)
+	local xstep = (1024/self.ScreenWidth)
+	local ystep = (1024/self.ScreenHeight)
```

Comment:
Why's the resolution getting doubled? It barely functions with 512 pixels

Reply:
because i have made the default RT size to be 1024x1024. i had to change every screen to comply. egp and gpu didnt get the upgrade and they are still at 512x512 with some uv magic.

Reply:
Is it possible to not upgrade digital screen too since it doesn't benefit at all from it.

Reply:
It technically does benefit, as resolutions which aren't a power of 2 will have more detail. But this could be solved now with the uv magic. Ill see what I can do.

Reply:
Oh, yeah I guess it does benefit for smaller resolutions. Maybe we can keep it then.

Reply:
Nice, thanks